collect_app/src/main/res/layout/project_list_item.xml

```diff
@@ -24,13 +27,15 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/margin_small"
-            android:textAppearance="@style/TextAppearance.Collect.Subtitle1" />
+            android:textAppearance="@style/TextAppearance.Collect.Subtitle1"
```

Comment:
These `textAppearance` attribute values should use the theme attributes instead.

Reply:
Made this change. Note it's not something I touched here, the diff was just the `/>`. I did have a related question at #4601 where I used theme attributes even though it went against the established patterns in the file I was in.

I've left the `@style/TextAppearance.Collect.Body2.MediumEmphasis` below for now since there isn't a corresponding theme attribute and the style is used in a few places. There isn't a lot of consistency across layouts currently and I think it's something that can be better addressed in a separate pass.